src/shim.c

```diff
@@ -331,17 +331,17 @@ send_connect_command(struct cc_shim *shim)
  *
  * \return true on success, false otherwise
  */
-bool read_wire_data(int fd, uint8_t *buf, ssize_t size)
+bool read_wire_data(int fd, uint8_t *buf, size_t size)
 {
 	ssize_t ret;
-	ssize_t offset = 0;
+	size_t offset = 0;
 
-	if ( fd < 0 || ! buf ) {
+	if ( fd < 0 || ! buf) {
```

Comment:
This is a spurious white space changes. The rule is no such thing sharing the same commit with other changes. The indentation in the shim is a bit all over the place, seems like we have hard time decided what should be used:

```
if(foo)
if (foo)
if ( foo )
```
and now:
```
if ( foo)
```

Life may well be too short for this, but opened https://github.com/clearcontainers/shim/issues/31 anyway. This isn't high priority.